### PR TITLE
Feature/tariik mvp/assign developers/ValidationToken

### PR DESCRIPTION
**Pull request by TarikVelic**
--------------------------------------------------------------------
 MAIN
 * Added employmentType (full-time, part-time)
 * Added token for validation if token exist or not

--------------------------------------------------------------------
 OTHER
 * <other msg> --------------------------------------------------------------------

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -37,6 +38,9 @@ app.get('/', (req, res) => {
   res.send('Hello World!');
 });
 
+// Ruta za provjeru valjanosti tokena
```
Comment (becarselman) [inline app.js]: Please use only English

File: controllers/validateToken.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+const tokenService = require('../services/validateToken');
+
+function validateToken(req, res) {
+  const { token } = req.body;
+
+  const isValid = tokenService.validateToken(token);
```
Comment (becarselman) [inline controllers/validateToken.js]: this need to be done in service.

File: services/validateToken.js
Comment (becarselman) [inline services/validateToken.js:1]: Idk is this pr wip or you didn't understand your task.

**Review by becarselman — CHANGES_REQUESTED** [review]

File: services/validateToken.js
Comment (becarselman) [inline services/validateToken.js:1]: the point of this task is to check if the jwt is valid or not. This will not help us in solving that problem.

**Review by becarselman — CHANGES_REQUESTED** [review]

File: services/validateToken.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const jwt = require('jsonwebtoken');
+
+const secretKey = process.env.JWT_SECRET;
+
+function validateToken(token) {
+  try {
+    jwt.verify(token, secretKey);
```
Comment (becarselman) [inline services/validateToken.js]: we can reuse function from utils/helpers that we use in past. And also I this this function will return true and false so we don't need to create condition we just return what ever this function return. But test this

**Review by becarselman — CHANGES_REQUESTED** [review]

File: controllers/validateToken.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+const tokenService = require('../services/validateToken');
+
+function validateToken(req, res) {
+  const { token } = req.body;
+  const isValid = tokenService.validateToken(token);
+
+  if (isValid) {
```
Comment (becarselman) [inline controllers/validateToken.js]: this need to be done in service. I think validate token will return true and false so you don't need this condition

**Review by becarselman — CHANGES_REQUESTED** [review]

**Review by AntMarkoLovric — APPROVED** [review]

**Pull request closed without merging**